src/shared/common/forms/mapped-display.jsx

```diff
@@ -10,26 +10,35 @@ class MappedDisplay extends Component {
     render() {
         const {
             choices,
+            component,
             label,
             value
         } = this.props;
         const mappings = new Map(choices.map(choice => [choice.value, choice.label]));
-        return (
-            <span className='govuk-body full-width'><strong>{label}: </strong>{mappings.get(value) ? mappings.get(value) : value}</span>
-        );
+        if (component === 'checkbox') {
+            return (
+                <span className='govuk-body full-width'><strong>{choices[0].label}: </strong><input className="bigger" type="checkbox" disabled="disabled" checked="checked"/></span>
```

Comment:
I know nothing of this ticket, but generally things like 
```html
<strong>Foo:</strong> bar
```
are better written using the `dd` and `dt` elements.

Reply:
strong used in other components. I'd rather stay with the standard.